src/IssueViz/Editor/QuickActions/QuickFixes/QuickFixSuggestedAction.cs

```diff
@@ -52,7 +52,8 @@ internal class QuickFixSuggestedAction : BaseSuggestedAction
 
         public override void Invoke(CancellationToken cancellationToken)
         {
-            if (cancellationToken.IsCancellationRequested)
+            if (cancellationToken.IsCancellationRequested ||
+                !quickFixVisualization.IsApplicable(textBuffer.CurrentSnapshot))
```

Comment:
Is not doing anything here a good solution? I think we should somehow inform the user that quick fix is not applicable anymore.

Reply:
I'll add logging to the output window so we would at least know it happened. I think a message box might be too intrusive, I'm not sure what would be the best UX. There is also a status bar in VS but no one ever notices it.